### PR TITLE
[DS-420] Add the y_camp module to composer;  Added Field Formatter to display referenced menu

### PR DESCRIPTION
**Pull request by aleevas**

### CODE REVIEW

File: src/Plugin/Field/FieldFormatter/EntityReferenceMenuFormatter.php [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+ * @FieldFormatter(
+ *   id = "entity_reference_menu_view",
+ *   label = @Translation("Rendered menu"),
+ *   description = @Translation("Display the referenced menu"),
+ *   field_types = {
+ *     "entity_reference"
+ *   }
+ * )
+ */
+class EntityReferenceMenuFormatter extends EntityReferenceFormatterBase {
```
Comment (podarok) [inline src/Plugin/Field/FieldFormatter/EntityReferenceMenuFormatter.php:26]: this looks like a Formatter to be placed into menu related module, maybe in inline_entity_form as a submodule or openy_er module, where we do have Entity Reference Enxhancements

Comment (anpolimus) [reply src/Plugin/Field/FieldFormatter/EntityReferenceMenuFormatter.php:26]: I agree, that this also should go to the IEF contrib submosule

Comment (podarok) [reply src/Plugin/Field/FieldFormatter/EntityReferenceMenuFormatter.php:26]: this one https://git.drupalcode.org/project/inline_entity_form/-/tree/2.0.x/modules/inline_entity_menu_form @aleevas

Comment (aleevas) [reply src/Plugin/Field/FieldFormatter/EntityReferenceMenuFormatter.php:26]: @anpolimus @podarok Here is the MR to the IEF module https://git.drupalcode.org/project/inline_entity_form/-/merge_requests/85

**Pull request merged**